T2Terminal/TerminalGlass.cpp

```diff
@@ -413,8 +413,8 @@ void Rushabh::TerminalGlass::Render(HDC hdc, struct Attributes attributes)
 			modelMatrix = translate(attributes.translateCoords[0] + translateCoords[i][0],
 									attributes.translateCoords[1] + translateCoords[i][1]+y,
 									attributes.translateCoords[2] + translateCoords[i][2]);			
-			//rotateMatrix = rotate(-30.0f, 1.0f, 0.0f, 0.0f);
-			//modelMatrix = modelMatrix * rotateMatrix;
+			rotateMatrix = rotate(90.0f, 1.0f, 0.0f, 0.0f);
```

Comment:
Don't use rotate call. Because when we would use lookat it would create problems. 
So its better to adjust the coordinates accordingly that you won't need to rotate the quad.